src/mcp-bundle/src/Routing/RouteLoader.php

```diff
@@ -11,27 +11,43 @@
 
 namespace Symfony\AI\McpBundle\Routing;
 
+use Symfony\AI\McpBundle\Exception\LogicException;
+use Symfony\Component\Config\Loader\Loader;
 use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 
-final readonly class RouteLoader
+final class RouteLoader extends Loader
```

Comment:
```suggestion
final readonly class RouteLoader extends Loader
```

Reply:
@OskarStark I cannot add `readonly` to the class because a readonly class cannot extend a non-readonly class (`Loader`). This is a PHP limitation. Should I keep only
  `final class RouteLoader extends Loader`?